<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Request</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Url</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="urlEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="methodCombo">
           <item>
            <property name="text">
             <string>GET</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>POST</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PUT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DELETE</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="requestButtons">
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>History</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="historyTable"/>
          </item>
          <item>
           <widget class="QToolButton" name="removeHistoryButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="remove">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Body</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPlainTextEdit" name="bodyText"/>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="bodyButtons">
         <property name="standardButtons">
          <set>QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Response</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="responseText"/>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="responseButtons">
         <property name="standardButtons">
          <set>QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
